bsip-0075.md

```diff
@@ -15,28 +15,85 @@ Worker: TBD
+
+
+## <div id="permissions">Permission to Change the Parameters</div>
+
+The permission to change any of these parameters shall make use of new permissions.  _Each of the parameters shall have its own distinct permission._  Disabling the permission prohibits the changing of the parameter as described in [Changing the Parameters](#changing-parameters).
 
-The `create_asset_operation` shall be modified to include the setting of the parameters above, as well as initial `flags` values and ratios for MCR and MSSR.
+The existing set of asset issuer permissions shall be expanded to permit an asset owner to forfeit changing the parameters.  It shall not be permitted to regain a permission after it is forfeited.
 
-The `update_asset_operation` shall be modified to permit updating of the MCR and MSSR values, if the asset has the appropriate `flags` value set permitting the asset owner to do so. NOTE: much of the logic currently in `asset_publish_feeds_evaluator::do_apply()` (i.e. call order checks and calculations) shall be also called here if MCR or MSSR are adjusted.
+Existing software clients might not be updated to make use of these new parameters.  Changing the permissions shall be designed such that older software clients may not accidentally forfeit these permissions either for existing assets or new assets.  _The parameters and the permissions to change parameters shall be designed such that they are defaulted to enabled.  The permissions shall be designed such that **explicit disabling/forfeiting** is required which should not be possible with older software clients._
 
-The `asset_update_bitasset_operation` shall be modified to permit updating of the MCR and MSSR values, if the asset has the appropriate `flags` value set permitting the asset owner to do so. NOTE: much of the logic currently in `asset_publish_feeds_evaluator::do_apply()` (i.e. call order checks and calculations) shall be called here if MCR and MSSR are adjusted.
+**Implementation Hints**
+
+The `asset_object.asset_options.issuer_permissions` may be expanded to include `disable_mcr_update`, `disable_mssr_update`.
+
+Appropriate changes to the ASSET_ISSUER_PERMISSION_MASK and UIA_ASSET_ISSUER_PERMISSION_MASK are advised to be both backward and forward compatible.
+
+
+## <div id="changing-parameters">Changing the Parameters</div>
+
+It shall be possible to set and unset a parameter at any time __if and only if__ its respective [permission](#permissions) is enabled.
+
+**Implementation Hints**
+
+The `create_asset_operation` can be modified to include the setting of the parameters above.
+
+The `update_asset_operation` can be modified to permit updating of the MCR and MSSR parameters, if the asset has the appropriate permissions.  Much of the logic currently in `asset_publish_feeds_evaluator::do_apply()` (i.e. call order checks and calculations) can be also called here if MCR or MSSR are adjusted.
```

Comment:
For this BSIP, `update_asset_operation` is only used to forfeit the permissions (and to regain if supply is zero).